js_modules/dagit/packages/core/src/assets/groupByPartition.tsx

```diff
@@ -23,10 +23,7 @@ const sortByEventTimestamp = (a: Event, b: Event) => Number(b?.timestamp) - Numb
  * A hook that can bucket a list of materializations by partition, if any, with the `latest`
  * materialization separated from predecessor materializations.
  */
-export const groupByPartition = (
-  events: Event[],
-  definedPartitionKeys: string[],
-): AssetEventGroup[] => {
+const groupByPartition = (events: Event[], definedPartitionKeys: string[]): AssetEventGroup[] => {
```

Comment:
Kinda weird this is the namesake function of the file and isn't exported 👀 I wonder if even more is unused here?